src/logic/calculate.js

```diff
@@ -9,22 +9,22 @@ const calculate = (data, buttonName) => {
   if (digits.includes(buttonName)) {
```

Comment:
I think `.` should be handled a little different from the digits. You should prevent the use of `.` more than once. See below:

![caldoterr](https://user-images.githubusercontent.com/36057474/73157805-836d8200-40e2-11ea-96fe-8bc62fb752a8.gif)